### PR TITLE
[Newton Compute AG: CWP-29368]: Support RedHat non-RPM packages

### PR DESCRIPTION
**Pull request by divyabhushan-okta**
## Reference

https://redlock.atlassian.net/browse/CWP-29368

## Description

Verified by PMs

### CODE REVIEW

File: compute/admin_guide/vulnerability_management/prisma_cloud_vulnerability_feed.adoc [hunk 1]
```diff
@@ -52,15 +52,20 @@ This will likely be an external advisory, a GitHub (or another bug tracker) issu
...
+[NOTE]
+====
+* For an application that originates from an OS package, the vulnerability data for CVEs are sourced from the relevant feed for the OS package. In some cases, like with Amazon Linux and Photon OS, this CVE information is provided in security advisories such as Amazon Linux Security Advisories (ALAS) for Amazon, and PHSA for Photon. In such cases, the correlation for the relevant vulnerabilities is limited. As an example, when the application "Python" is sourced from an Amazon Python package, CVEs found for the python application (as a binary) will not be correlated with the relevant Amazon CVEs from the ALAS.
+
+* Prisma Cloud Intelligence Stream uses RedHat's image analysis to accurately flag the vulnerable images with the same severity as detected in https://catalog.redhat.com/software/containers/search[RedHat Ecosystem Catalog].
```
Comment (divyabhushan-okta) [inline compute/admin_guide/vulnerability_management/prisma_cloud_vulnerability_feed.adoc]: @tallieber FYI

**Review by divyabhushan-okta — COMMENTED** [review]
Update Red Hat analysis data

**Comment by rcaballeromx** [comment]
@divyabhushan Is this content ready for merging?

**Comment by divyabhushan-okta** [comment]
> @divyabhushan Is this content ready for merging?

@rcaballeromx Yes, this PR is ready to be merged.

**Review by rcaballeromx — APPROVED** [review]

**Comment by rcaballeromx** [comment]
Thanks @divyabhushan, I've merged the content.

**Pull request merged**